shell/shell.go

```diff
@@ -61,6 +61,12 @@ func ExecuteShell(uid uint32,
 	cmd.Env = append(cmd.Env, fmt.Sprintf("HOME=%s", homeDir))
 	cmd.Env = append(cmd.Env, fmt.Sprintf("TERM=%s", termString))
 
+	log.Info("using script command to start shell")
+	cmd_wrapper := exec.Command("script", "-qec", shell, "/dev/null")
+	cmd_wrapper.SysProcAttr = cmd.SysProcAttr
+	cmd_wrapper.Dir = cmd.Dir
+	cmd_wrapper.Env = cmd.Env
+	cmd = cmd_wrapper
```

Comment:
This is discarding the initialization of the cmd above...
On the other hand, could this be solved by a simple config default change?
You could try adding the following to `/etc/nt-connect/nt-connect.json`:
```json
{
  "ShellCommand": "script",
  "ShellArguments": ["-qec", "/bin/bash", "/dev/null"]
}
```